CMakeLists.txt

```diff
@@ -40,6 +40,10 @@ set(LLVM_ENABLE_TERMINFO OFF CACHE BOOL "")
 # LLVM settings that have an effect on the MLIR dialect
 set(LLVM_TARGETS_TO_BUILD "X86;AMDGPU" CACHE STRING "")
 
+if (MLIR_INCLUDE_TESTS)
```

Comment:
I think we end up having to use -DMLIR_INCLUDE_TESTS to get --tosa-partition-pipeline in our mlir-opt, though not in the external mlir-opt.